src/lib/DefaultRuleManager.ts

```diff
@@ -68,11 +66,15 @@ export class DefaultRuleManager implements RuleManager {
 
 		// Execute all run promises, each of which returns an array of RuleResults, then concatenate
 		// all of the results together from all engines into one report.
-		const psResults: RuleResult[][] = await Promise.all(ps);
-		psResults.forEach(r => results = results.concat(r));
-		this.logger.trace(`Received rule violations: ${results}`);
-		this.logger.trace(`Recombining results into requested format ${format}`);
-		return RuleResultRecombinator.recombineAndReformatResults(results, format);
+		try {
+			const psResults: RuleResult[][] = await Promise.all(ps);
+			psResults.forEach(r => results = results.concat(r));
+			this.logger.trace(`Received rule violations: ${results}`);
+			this.logger.trace(`Recombining results into requested format ${format}`);
+			return RuleResultRecombinator.recombineAndReformatResults(results, format);
+		} catch (e) {
+			throw new SfdxError(e.stack || e);
```

Comment:
Will we need to wrap the stack in something helpful to explain "Problem occurred while trying to do X, here's the stack"?

Reply:
Oops #2.  I didn't mean to check that in, just changed it off the cuff to see the actual error info.

Reply:
Should this still be here?

Reply:
Gone.